terraform/variables.tf

```diff
@@ -71,3 +71,21 @@ locals {
 
   azure_credentials = try(jsondecode(var.azure_credentials), null)
 }
+
+# Azure Resource Variables
+
+variable resource_group_name {}
+
+variable azure_resource_group_name {}
+
+variable azure_resource_group_tags { default = "" }
+
+variable azure_storage_diagnostics_logging_enabled { default = false }
```

Comment:
We don't log at the moment, can we add it only when we need it?

Reply:
Removed